docs/development/shareable-configs.md

```diff
@@ -11,105 +11,32 @@ Unlike ESLint though:
-
-Note: if using non-scoped packages, a preset name is mandatory.
-
-```
-renovate-config-somepackagename:default
-```
-
-This will resolve to use the preset `default` in the npm package `renovate-config-somepackagename`.
-
-### Non-scoped full with preset name and params
-
-Note: if using non-scoped packages, a preset name is mandatory.
-
-```
-renovate-config-somepackagename:default(eslint)
-```
-
-This will resolve to use the preset `default` in the npm package `renovate-config-somepackagename` and param `eslint`.
+| name                                               | example use                                 | preset    | npm package resolves as      | parameters |
+| -------------------------------------------------- | ------------------------------------------- | --------- | ---------------------------- | ---------- |
+| scoped                                             | `@somescope`                                | `default` | `@somescope/renovate-config` |            |
+| scoped with package name                           | `@somescope/somepackagename`                | `default` | `@somescope/somepackagename` |            |
+| scoped with preset name                            | `@somescope:webapp`                         | `webapp`  | `@somescope/renovate-config` |            |
+| scoped with param                                  | `@somescope(eslint)`                        | `default` | `@somescope/renovate-config` | `eslint`   |
+| scoped with preset name and param                  | `@somescope:webapp(eslint)`                 | `webapp`  | `@somescope/renovate-config` | `eslint`   |
+| scoped with package name and preset name           | `@somescope/somepackagename:webapp`         | `webapp`  | `@somescope/somepackagename` |            |
+| scoped with package name and preset name and param | `@somescope/somepackagename:webapp(eslint)` | `webapp`  | `@somescope/somepackagename` | `eslint`   |
+
+### Non-scoped
+
+If you use a non-scoped config, you must use a preset name!
+
+| name                                        | example use                                       | preset    | npm package resolves as           | parameters |
+| ------------------------------------------- | ------------------------------------------------- | --------- | --------------------------------- | ---------- |
+| non-scoped short with preset name           | `somepackagename:default`                         | `default` | `renovate-config-somepackagename` |            |
+| non-scoped short with preset name and param | `somepackagename:default(eslint)`                 | `default` | `renovate-config-somepackagename` | `eslint`   |
+| non-scoped full with preset name            | `renovate-config-somepackagename:default`         | `default` | `renovate-config-somepackagename` |            |
+| non-scoped full with preset name and param  | `renovate-config-somepackagename:default(eslint)` | `default` | `renovate-config-somepackagename` | `eslint`   |
```

Comment:
The current table is a bit too wide to review comfortably in a GitHub PR screen, can we maybe trim some content somewhere? Or would trimming even more make this hard to understand for readers?

Also, are you happy with the ordering of the content? I kept the original order, but maybe we're better of switching some content around? To maybe make the `preset` column have `default` first, and then `webapp`?

I'll stay away from changing this branch so that you have the opportunity to make the changes yourself without me getting in the way. :smile:

Reply:
I think the width is fine. These docs aren't for sharing in PRs anyway, and their target location is our docs page. Let's merge as-is